modules/microshift-exposed-audit-ports-hostnetwork.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-exposed-audit-ports-hostnetwork_{context}"]
+
+== hostnetwork
+
+When a pod is configured with the `hostNetwork:true` setting, the pod is running in the host network namespace. This configuration can independently open host ports. {microshift-short} component logs cannot be used to track this case, the ports are subject to firewalld rules. If the port opens in firewalld, you can view the port opening in the firewalld debug log.
+
+.Prerequisites 
+
+* You have root user access to your build host.
+
+.Procedure
+
+. Enable debug in the firewalld log by running the following command:
++
+[source,terminal]
+----
+$ sudo vi /etc/sysconfig/firewalld
+FIREWALLD_ARGS=--debug=10
+----
+
+. Restart the firewalld service
++
+[source,terminal]
+----
+$ sudo systemctl restart firewalld.service
+----
++
+.Example logs for when the port open rule is added:
```

Comment:
How are these example generated ? Are they in a log file ?

Reply:
Hmm, I've been looking around for this one and I'm not sure. @pliurh Do you know this one?

Reply:
The firewalld debug log is stored at `/var/log/firewalld`